.github/workflows/test.yml

```diff
@@ -29,4 +29,28 @@ jobs:
     - name: Test with module pytest
       run: |
         python -m pytest tests
-
+  test-install:
+    name: Test import of packages after local install
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Update pip
+        run: python3 -m pip install --upgrade pip
+      - name: Install pytest
+        run: python -m pip install pytest
+      - name: Get YaLafi (in subdirectory to check actual installation)
+        run: git clone https://github.com/torik42/YaLafi
+      - name: Install YaLafi
+        run: |
+          cd YaLafi
+          pip install .
```

Comment:
This is what I meant before. In the next PR, we should run the install with the PR branch, not with a fresh clone of the master. That’s why we should stick to `actions/checkout@master` which will automatically checkout the PR.

Reply:
You are correct. 
I actually deleted that `checkout` part intentionally earlier. This was to have the repository in a subdirectory, so that the `import` cannot load the local files.
How do I ensure this then? `cd` into `tests` or to the parent (not sure that works) before trying the import?

Reply:
Good point. As far as I understand [this](https://github.com/actions/checkout#usage),
```
with:
   path: …
```
should do the trick.

Reply:
Alternatively, we could delete the directory again.

Reply:
Thanks for pointing me to this, seems to work well.